pkgs/by-name/cy/cyrus_sasl/package.nix

```diff
@@ -74,7 +74,7 @@ stdenv.mkDerivation rec {
       "--enable-shared"
     ]
     ++ lib.optional enableLdap "--with-ldap=${openldap.dev}"
-    ++ lib.optionals (stdenv.targetPlatform.useLLVM or false) [
+    ++ lib.optionals (stdenv.cc.isClang && !stdenv.hostPlatform.isDarwin) [
```

Comment:
Should this be checking `hostPlatform.cc == "clang"` instead of `stdenv.cc.isClang`?

Reply:
Yes, I just made this change before running into the `stdenv.cc.is*` problem.